src/components/PageBlocks/CalendarBase.vue

```diff
@@ -5,7 +5,10 @@
     v-on="$listeners"
   >
     <div class="calendar-container m-2">
-      <div class="d-flex align-items-baseline justify-content-center mb-2">
+      <div
+        v-if="!header.hidePrevNext && !header.hide"
```

Comment:
I reccomend you wrap the prev/next annd the view switcher into a div so you need to check !header.hide only once